test/parallel/test-require-json.js

```diff
@@ -24,10 +24,9 @@ require('../common');
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
 
-try {
+assert.throws(function() {
   require(fixtures.path('invalid.json'));
-} catch (err) {
-  assert.ok(
-    /test[/\\]fixtures[/\\]invalid\.json: /.test(err.message),
-    `require() json error should include path: ${err.message}`);
-}
+}, {
+      name: 'SyntaxError',
```

Comment:
The linter failed because the expected indentation here is two spaces more than the previous line. Can you adjust and run `make lint-js` (or `vcbuild lint-js` on Windows) to make sure all linter stuff is handled, then push to your branch?

Reply:
Fixed in edac77982b56b5a358361048ad6166e7f2f1c725